docs/en/observability/uptime-set-up-app.asciidoc

```diff
@@ -0,0 +1,168 @@
+[[uptime-set-up-choose-agent]]
+= Use the {uptime-app}
+
+beta[]
+
+You can now create monitors directly in the {uptime-app}.
+To try out this beta functionality, first you will need to enable Monitor Management:
+
+. Go to **{observability}** -> **Uptime**.
+. Click **Monitor Management**.
+. Review the terms and click **Enable** to start using Monitor Management.
```

Comment:
NIT: doesn't make a huge difference, but "review the terms" sounds a bit weird as I always read "terms and conditions" elsewhere.

```suggestion
. Review the terms and conditions and click **Enable** to start using Monitor Management.
```